src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -60,6 +60,7 @@ const propTypes = {
 
     isSidebarAnimating: PropTypes.bool,
     isChatSwitcherActive: PropTypes.bool,
+
```

Comment:
Still has an unecessary change 🙃

Reply:
haha. You are a good reviewer after all. 💂

Reply:
This way, I will learn etiquettes. Thanks.